docs/source/ux_limitations.rst

```diff
@@ -179,3 +179,45 @@ JAX supports transforming over
 using special control flow operators (e.g. ``jax.lax.cond``, ``jax.lax.while_loop``).
 We're investigating adding equivalents of those to functorch
 (open an issue on `GitHub <https://github.com/pytorch/functorch>`_ to voice your support!).
+
+Randomness
+----------
+Note that like with JAX's PRNG system, our system only determine the randomness behavior of PyTorch
+operators
+
+Our randomness differs from JAX because we aren't using a stateless PRNG, in part because PyTorch doesn't
+have full support for a stateless PRNG. Instead, we've introduced a flag system to allow for the most
+common forms of randmoness that we see. If your use case does not fit these forms of randomness, please
+file an issue.
+
+The flag can only be passed to vmap and can take on 3 values, "error," "different," or "same," defaulting
+to error. Under "error" mode, any call to a random function will produce an error asking the user to use
+one of the other two flags based on their use case.
+
+Under "different" randomness, elements in a batch produce different random values. For instance,
+
+::
+
+  def add_noise(x):
+    y = torch.randn(4)  # y will be different across the batch
+    return x + y
+
+  x = torch.ones(3, 4)
+  result = vmap(add_noise, randomness="different")(x)
+  assert result[0] != result[1]
```

Comment:
would this be easier to understand if we did fewer operations?
```
def f(x):
  return torch.rand([])

result = vmap(f, randomness='different')(torch.ones(3))
# we get three different random numbers!
```
The same randomness case is, we get 3 same random numbers

Reply:
Agreed. Changed